cookbooks/aws-parallelcluster-install/recipes/install_mysql_client.rb

```diff
@@ -24,7 +24,7 @@
 
   package_installer = value_for_platform(
     'default' => "yum install -y",
-    'ubuntu' => { 'default' => "apt install -y" }
+    'ubuntu' => { 'default' => "apt install -y --allow-downgrades" }
```

Comment:
does the new flag "--allow-downgrades" added for apt needs to be aligned also for yum?